schemas/bedrock-kms-http.js

```diff
@@ -0,0 +1,192 @@
+    }
+  }
+};
+
+const findKeystore = {
+  title: 'findKeystore',
+  type: 'object',
+  additionalProperties: false,
+  required: ['controller', 'referenceId'],
+  properties: {
+    controller,
+    referenceId
+  }
+};
+
+const zcap = {
+  title: 'zcap',
+  type: 'object',
+  additionalProperties: false,
+  required: [
+    'id', 'invoker', 'parentCapability', 'allowedAction', 'invocationTarget'
+  ],
+  properties: {
+    controller,
+    invoker,
+    delegator,
+    id: {
+      title: 'id',
+      type: 'string'
+    },
+    allowedAction: {
+      anyOf: [{
+        type: 'string'
+      }, {
+        type: 'array',
+        minItems: 1,
+        items: {type: 'string'}
+      }]
+    },
+    caveat: {
+      title: 'Caveat',
+      type: 'object'
+    },
+    '@context': {
+      title: '@context',
+      anyOf: [{
+        type: 'string'
+      }, {
+        type: 'array',
+        minItems: 1,
+        items: {type: 'string'}
+      }]
+    },
+    invocationTarget: {
+      title: 'Invocation Target',
+      anyOf: [{
+        type: 'string'
+      }, {
+        type: 'object',
+        required: [
+          'type', 'id'
+        ],
+        additionalProperties: false,
+        properties: {
+          id: {
+            title: 'Invocation Target Id',
+            type: 'string'
+          },
+          type: {
+            title: 'Invocation Target Type',
+            type: 'string'
+          },
+          controller: {
+            title: 'controller',
+            type: 'string'
+          },
+          verificationMethod: {
+            title: 'verificationMethod',
+            type: 'string'
+          }
+        }
+      }]
+    },
+    parentCapability: {
+      title: 'Parent Capability',
+      type: 'string'
+    },
+    proof: {
+      title: 'Proof',
+      type: 'object',
+      additionalProperties: false,
+      properties: {
+        verificationMethod: {
+          title: 'verificationMethod',
+          type: 'string'
+        },
+        type: {
+          title: 'type',
+          type: 'string'
+        },
+        created: {
+          title: 'created',
+          type: 'string'
+        },
+        proofPurpose: {
+          title: 'proofPurpose',
+          type: 'string'
+        },
+        capabilityChain: {
+          title: 'capabilityChain',
+          type: 'array',
+          minItems: 1,
+          items: {
+            type: ['string', 'object']
+          }
+        },
+        jws: {
+          title: 'jws',
+          type: 'string'
+        },
+      }},
+    referenceId
+  }
+};
+
+const recovery = {
+  title: 'recovery',
+  type: 'object',
+  additionalProperties: false,
+  required: ['@context', 'controller'],
+  properties: {
+    controller,
+    '@context': {
+      title: '@context', type: 'string'
+    }}
+};
+
+module.exports = {
+  findKeystore: () => findKeystore,
+  postKeystore: () => postKeystore,
+  zcap: () => zcap,
+  recovery: () => recovery
```

Comment:
what convention are we using for this?

As we start tightening things up around validation...  The schema exports of a bedrock module are actually part of its public interface, and therefore we need to start tracking changes to validators carefully.  Changing the name of a validator and certain changes to the schema itself later will represent a breaking change.

Also, now that the validators are becoming more sophisticated, we want to reduce duplication across repos, and therefore we want to start leveraging the public exports from other modules.

Reply:
This is the convention I have seen used in every validation schema that I have come across. [Example](https://github.com/digitalbazaar/bedrock-account-http/blob/master/schemas/account.js#L66) and [example](https://github.com/digitalbazaar/bedrock-stats-http/blob/master/schemas/stats-http.js#L29). I think finding a more centralized place for all of the re-usable schemas would solve this problem. Is that something worth adding to the [bedrock-validation](https://github.com/digitalbazaar/bedrock-validation/tree/master/schemas) module itself, or splitting it into specific modules, like the `bedrock-zcap` mentioned in the previous comment?

Reply:
Sorry, I should have said "naming convention".  We had a discussion last week about some convention for endpoints that involved something like - method + foo

Reply:
[Here](https://github.com/digitalbazaar/bedrock-edv-storage/pull/64#discussion_r452428611) is the previous conversation. I have updated the names of these validators according to the previous comments. I have kept `zcap` the same for the time being.